### ISSUE TITLE
Unable to instantiate application chat.rocket.reactnative.MainDebugApplication

### ISSUE THREAD

**Issue by VinodComlink** [RocketChat/Rocket.Chat.ReactNative#4980]
### Description:

Android app crashing on launch with below Exception -

```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: chat.rocket.reactnative, PID: 31322
    java.lang.RuntimeException: Unable to instantiate application chat.rocket.reactnative.MainDebugApplication package chat.rocket.reactnative: java.lang.ClassNotFoundException: Didn't find class "chat.rocket.reactnative.MainDebugApplication" on path: DexPathList[[zip file "/data/app/~~e23JEYRqxJFtsvV_dWkfMA==/chat.rocket.reactnative-Yj1F3egTkptRni49yit0nQ==/base.apk"],nativeLibraryDirectories=[/data/app/~~e23JEYRqxJFtsvV_dWkfMA==/chat.rocket.reactnative-Yj1F3egTkptRni49yit0nQ==/lib/arm64, /data/app/~~e23JEYRqxJFtsvV_dWkfMA==/chat.rocket.reactnative-Yj1F3egTkptRni49yit0nQ==/base.apk!/lib/arm64-v8a, /system/lib64, /system_ext/lib64]]
        at android.app.LoadedApk.makeApplication(LoadedApk.java:1377)
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7063)
        at android.app.ActivityThread.access$1600(ActivityThread.java:271)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2134)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loopOnce(Looper.java:210)
        at android.os.Looper.loop(Looper.java:299)
        at android.app.ActivityThread.main(ActivityThread.java:8319)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:556)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1038)
     Caused by: java.lang.ClassNotFoundException: Didn't find class "chat.rocket.reactnative.MainDebugApplication" on path: DexPathList[[zip file "/data/app/~~e23JEYRqxJFtsvV_dWkfMA==/chat.rocket.reactnative-Yj1F3egTkptRni49yit0nQ==/base.apk"],nativeLibraryDirectories=[/data/app/~~e23JEYRqxJFtsvV_dWkfMA==/chat.rocket.reactnative-Yj1F3egTkptRni49yit0nQ==/lib/arm64, /data/app/~~e23JEYRqxJFtsvV_dWkfMA==/chat.rocket.reactnative-Yj1F3egTkptRni49yit0nQ==/base.apk!/lib/arm64-v8a, /system/lib64, /system_ext/lib64]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at android.app.AppComponentFactory.instantiateApplication(AppComponentFactory.java:76)
        at androidx.core.app.CoreComponentFactory.instantiateApplication(CoreComponentFactory.java:52)
        at android.app.Instrumentation.newApplication(Instrumentation.java:1178)
        at android.app.LoadedApk.makeApplication(LoadedApk.java:1369)
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7063) 
        at android.app.ActivityThread.access$1600(ActivityThread.java:271) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2134) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loopOnce(Looper.java:210) 
        at android.os.Looper.loop(Looper.java:299) 
        at android.app.ActivityThread.main(ActivityThread.java:8319) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:556) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1038) 

```

### Environment Information:

- Rocket.Chat Server Version:
- Rocket.Chat App Version:  4.36.0
- Device Name: RealME 
- OS Version: 11

### Steps to reproduce:

1. Install the dependencies 
2. Sync gradle 
3. Run the app 

### Expected behavior:

App Should launch without any issue 

### Actual behavior:

Crashing on launch

**Comment by VinodComlink** [comment]
Why it is closed ?

**Comment by GleidsonDaniel** [comment]
@VinodComlink you created countless different issues, each with a different problem.
Almost all of them apparently are problems with your environment, given that it's not just you using the app, there are several people and practically none have the same problem as yours, including new people who are opening PRs here in the repo or opening issues.
It would be simpler if you put together all your problems in a single issue or ask for help on the channel that is specific for this (https://open.rocket.chat/channel/react-native) and maybe even explain what you want to do, why, for example, in this PR here you said you wanted to build an old version of the app:
https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/4976

In this other the problem was related to the permissions of your pods:
https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/4959#issuecomment-1463742467
Then the installation of your pods was with error:
https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/4959#issuecomment-1464883536

So in summary, so far, all open issues are not related to the app but to wrong settings or something with your environment.

So in the next issues, explain the problem better, explain what you are trying to do, the steps you have already taken to solve the problem and if none of your attempts have worked, open an issue.
